src/components/WalletDropdown/MiniPortfolio/PortfolioLogo.tsx

```diff
@@ -60,31 +60,20 @@ const ENSAvatarImg = styled.img`
 const StyledChainLogo = styled.img<{ isSquare: boolean }>`
   height: ${({ isSquare }) => (isSquare ? '16px' : '14px')};
   width: ${({ isSquare }) => (isSquare ? '16px' : '14px')};
-  margin-top: ${({ isSquare }) => (isSquare ? '0px' : '1px')};
-  margin-left: ${({ isSquare }) => (isSquare ? '0px' : '1px')};
-  position: absolute;
-  top: 68%;
-  left: 68%;
 `
 
-const ChainLogoSquareBackground = styled.div`
-  height: 18px;
-  width: 18px;
-  border-radius: 4px;
-  background-color: ${({ theme }) => theme.backgroundSurface};
+const L2LogoContainer = styled.div<{ nonSquareLogo?: boolean }>`
+  background-color: ${({ theme, nonSquareLogo }) => (nonSquareLogo ? theme.textPrimary : theme.backgroundSurface)};
+  border-radius: 2px;
+  height: 16px;
+  left: 60%;
```

Comment:
don't change this code off of this - but imo percentages are always hard to work with

Reply:
if you're passing in a non-native prop to an html component make sure to use the `$variable` notation